regparser/utils.py

```diff
@@ -31,3 +33,36 @@ def title_body(text):
 def flatten(list_of_lists):
     """List[List[X]] -> List[X]"""
     return sum(list_of_lists, [])
+
+letters = [chr(i) for i in range(97, 97 + 26)]
+
+
+def random_letters(length):
+
+    result = ''
+    for i in range(length):
+        result += choice(letters)
+    return result
+
+
+def set_of_random_letters(num_items, length):
```

Comment:
Do these two functions (`random_letters` and `set_of_random_letters`) do anything? They're imported in `api_writer`, but I don't see them being used.